willyserge/broadcaster
# #170023410 implementing backed feedback

#### What does this PR do?
  implementation of the feedback given on the backend
#### Description of Task to be completed?
- validating user password to include at least a number, an uppercase, and a lower case letter
#### How should this be manually tested?
- go to `postman` and make a` POST request` to `http://localhost: port number /api/v1/auth/signup`
- on the password field use a password that does not match the required pattern and check the error
#### What are the relevant pivotal tracker stories?
[#169860322 ](https://www.pivotaltracker.com/n/projects/2418444/stories/170023410)
